### PR TITLE
Create TravelingSalesManPoblem.java

### PR DESCRIPTION
**Pull request by Sujal1804**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [x] All new code is formatted with `clang-format -i --style=file path/to/your/file.java`

### CODE REVIEW

File: src/test/java/com/thealgorithms/dynamicprogramming/TravelingSalesManPoblem.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class TSPTest {
```
Comment (saahil-mahato) [inline src/test/java/com/thealgorithms/dynamicprogramming/TravelingSalesManPoblem.java:4]: It's better to use `@ParameterizedTest`. Also, it is better to test both the path and minimum cost for all cases.

**Review by saahil-mahato — CHANGES_REQUESTED** [review]

**Comment by siriak** [comment]
Please check why PR checks are failing
image
